# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/audioipc/audioipc/Cargo.toml
# Commit: 3939f37f4740
# Full Hash: 3939f37f47400a76a94962da4c38c24456952ee6
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2020-01-26 09:48:16
# Regressor Bug: 1590249
# File Overlap Count: 3
# Description:
#   Bug 1590249 - Update audioipc to 86d49ddf.  r=chunmin
#   
#   Minus audio_thread_priority 0.21, due to Android build breakage.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60615
# ==============================================================================

diff -r a6f08a30777b -r 3939f37f4740 media/audioipc/audioipc/Cargo.toml
--- a/media/audioipc/audioipc/Cargo.toml	Sat Jan 25 22:49:44 2020 +0000
+++ b/media/audioipc/audioipc/Cargo.toml	Sat Jan 25 22:50:52 2020 +0000
@@ -29,9 +29,9 @@
 tokio-reactor = "0.1"
 
 [target.'cfg(windows)'.dependencies]
+mio = "0.6.19"
 miow = "0.3.3"
 mio-named-pipes = { git = "https://github.com/alexcrichton/mio-named-pipes" }
-tokio-named-pipes = { git = "https://github.com/NikVolf/tokio-named-pipes", branch = "stable" }
 winapi = { version = "0.3.6", features = ["combaseapi", "objbase"] }
 
 [dependencies.error-chain]